Bottom: 4ae9b262da8deec38620327533112e7829b84e56
Top:    06b3becd44c838ded926f8d4959e4610c00d0ef3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-01-06 16:47:37 +0000

Build with asserts for polygons


---

diff --git a/src/polygon.c b/src/polygon.c
index 75e1b9e..c0dee55 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -71,6 +71,7 @@ dicer output is used for HIDs which cannot render things with holes
 /* special polygon editing routines
  */
 
+#undef NDEBUG
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
diff --git a/src/polygon1.c b/src/polygon1.c
index 566596d..3d2e763 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,6 +39,7 @@
       are marked
 */
 
+#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
@@ -100,7 +101,7 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 #undef DEBUG_JUMP
 #undef DEBUG_GATHER
 #undef DEBUG_ANGLE
-#undef DEBUG
+#define DEBUG
 #ifdef DEBUG
 #define DEBUGP(...) pcb_fprintf(stderr, ## __VA_ARGS__)
 #else
